examples/run_examples.py

```diff
@@ -80,23 +80,25 @@ def check_device(rule, target):
 
 
 req_version = defaultdict(lambda:(2019,0))
+req_version['adaboost_batch.py'] = (2020,0)
+req_version['brownboost_batch.py'] = (2020,0)
+req_version['dbscan_batch.py'] = (2019,5)
 req_version['decision_forest_classification_batch.py'] = (2019,1)
 req_version['decision_forest_classification_traverse_batch.py'] = (2019,1)
 req_version['decision_forest_regression_batch.py'] = (2019,1)
-req_version['adaboost_batch.py'] = (2020,0)
-req_version['brownboost_batch.py'] = (2020,0)
+req_version['elastic_net_batch.py'] = ((2020,1),(2021,105))
+req_version['gbt_cls_model_create_from_lightgbm_batch.py'] = ((2020, 2), (2021, 109))
```

Comment:
This feature available from 2020u3 version?
```suggestion
req_version['gbt_cls_model_create_from_lightgbm_batch.py'] = ((2020, 3), (2021, 109))
```

Reply:
Yes, but tests are working on version (2020, 2), so our examples are being skipped when we use version (2020, 3)

Reply:
Ok